### PR TITLE
min-app

### PR DESCRIPTION
**Pull request by chobitsX**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.
- [x] it's declaration files for Weapp(weixin) mini program
- [x] So far the best preserved declaration files of typescript for mini program

### CODE REVIEW

**Review by andrewbranch — CHANGES_REQUESTED** [review]
Can you explain what this is?

**Comment by chobits4** [comment]
> Can you explain what this is?

- it's declaration files for Weapp(weixin) mini program
- So far the best preserved declaration files of typescript for mini program

**Review by chobitsX — COMMENTED** [review]
@AndrewBranch @typescript-bot 
- it's declaration files for Weapp(weixin) mini program
- So far the best preserved declaration files of typescript for mini program

File: types/min-app/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// Type definitions for min-app 1.0
+// Project: https://github.com/chobits4/min-app (Does not have to be to GitHub, but prefer linking to a source code repository rather than to a project website.)
+// Definitions by: My Self <https://github.com/chobits4>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.8
+
```
Comment (chobits4) [inline types/min-app/index.d.ts]: - it's declaration files for Weapp(weixin) mini program
- So far the best preserved declaration files of typescript for mini program

**Comment by chobits4** [comment]
> Can you explain what this is?

**Pull request closed without merging**